lib/classes/CLI.js

```diff
@@ -187,6 +188,20 @@ class CLI {
   consoleLog(message) {
     console.log(message); // eslint-disable-line no-console
   }
+
+  logBreakingChanges() {
```

Comment:
This should have a basic test (inspiration here: https://github.com/serverless/serverless/blob/master/lib/plugins/aws/info/display.test.js#L43-L60).